projects/uriparser/uri_dissect_query_malloc_fuzzer.cc

```diff
@@ -45,7 +45,9 @@ extern "C" int LLVMFuzzerTestOneInput(const uint8_t *data, size_t size) {
   int chars_required;
   if (uriComposeQueryCharsRequiredA(query_list, &chars_required) != URI_SUCCESS)
     return 0;
-    
+
+  if (!chars_required) return 0;
```

Comment:
do you need to call `uriFreeQueryListA` here?

Reply:
Good catch, thanks.